nekoyume/Assets/_Scripts/Extension/StatExtensions.cs

```diff
@@ -12,5 +13,34 @@ public static string DecimalStatToString(this DecimalStat stat)
 
             return $"{stat.Type} +{(float)value}";
         }
+
+        public static string OptionRowToString(this EquipmentItemOptionSheet.Row optionRow)
+        {
+            var statMin = ValueToString(optionRow.StatType, optionRow.StatMin);
+            var statMax = ValueToString(optionRow.StatType, optionRow.StatMax);
+
+            var description = $"{optionRow.StatType} +({statMin}-{statMax})";
+            return description;
+        }
+
+        public static string ValueToString(StatType statType, int value, string format = null)
```

Comment:
What do you think about changing it to `public static string ValueToString(this StatType statType, int value, string format = null)`

Reply:
Or how about this? There are many overridings.😁
```cs
public static string ValueToString(this (StatType statType, int value) tuple, string format = null) {}
```

Reply:
I agree because this class is for extension method. So I will choose `public static string ValueToString(this StatType statType, int value, string format = null)` form because not all stat values are contained in tuple.